rero/rero-ils-ui
# vendors: implements views

* Adds brief view.
* Adds detail view.
* Adds link to acquisition menu.

Co-Authored-by: Laurent Dubois <laurent.dubois@itld-solutions.be>

## Why are you opening this PR?

- Implements US1185: Organisation vendor

## How to test?

- This PR is based on rero/rero-ils/pull/668
- Login as librarian
- Check vendors section (Aquisitions menu entry) on admin view

## Code review check list
